ci/travis-before-install.sh

```diff
@@ -17,29 +17,27 @@ python -V
 
 pip install --upgrade pip setuptools
 
-if [ $TRAVIS_PULL_REQUEST = "false" ]
-then
-    branch=$TRAVIS_BRANCH
-else
-    branch=$TRAVIS_PULL_REQUEST_BRANCH
-fi
-
+# For push builds, TRAVIS_BRANCH is the branch name
+# For builds triggered by PR, TRAVIS_BRANCH is the branch targeted by the PR
+branch=$TRAVIS_BRANCH
 echo branch ${branch}
 
-
+# iterate over upstream dependencies
 for upstream_repo in vivarium vivarium_public_health
```

Comment:
I know we had this before but why does gbd_mapping rely on vivarium and vivarium_public_health?

Reply:
It doesn't.

Reply:
I'll remove